### ISSUE TITLE
Preparing for the PR against master

### ISSUE THREAD

**Issue by jasnell** [nodejs/http2#173]
This week I will be preparing the PR to open against nodejs/node master. Once that lands, all future work will be done in the nodejs/node repository following the normal Node.js core processes. In preparation for that, I will be squashing the commit history down to clean up the commit log. The PR is already going to be extremely difficult to review given the amount of code. The commit history in nodejs/http2 master *will not be changed* so we will still have a record of the individual contributions here.

Also, I'm currently working on rebasing nodejs/http2 master to nodejs/node master and will be force pushing soon. The current set of open PRs will need to be rebased if they have not landed before then. Let's try to get them landed soon! 

FYI: I will be on a plane Tuesday afternoon and most of the day Thursday but will be trying to work on the flight.

**Comment by jasnell** [comment]
All.. I am preparing to force push to master with the nodejs/node master rebase. Just FYI...

**Comment by sebdeckers** [comment]
@jasnell What's the process now that the PR is open? E.g. #170 has an update

Not sure how long the PR review might take. Could have a cooling-off period. Would prefer to continue collaborating in the mean time.

**Comment by jasnell** [comment]
I'll push the pr branch to nodejs/http2. We can open new prs against that :)

**Comment by sebdeckers** [comment]
Ah, that's great. I now see an `initial-pr` branch in this repo.

**Comment by sebdeckers** [comment]
Might also have worked if the big PR was from `nodejs/http2#pr` into `nodejs/node#master`.

**Comment by jasnell** [comment]
I had tried that but it's a bit difficult given that nodejs/http2 is not a true fork of the nodejs/node repo. As it is, any changes to the initial-pr branch in nodejs/http2 can be easily pushed to the initial-pr branch in jasnell/http2-1 so things should just work

**Comment by jasnell** [comment]
Going to close this issue now that the PR is opened!
